Evolve.Migrations.Helper/Program.cs

```diff
@@ -110,6 +116,12 @@ private static string GetFileSeparator(IReadOnlyList<string> args)
             }
         }
 
+        private static void PrintHeaderMessage()
+        {
+            Console.WriteLine("Evolve Migrations Helper");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/manual/odair-pedro/evolve-migrations-helper/pullRequest?prid=5618245)